recipe/run_test.sh

```diff
@@ -2,13 +2,11 @@
 
 set -e
 
```

Comment:
```suggestion

export LIBRARY_PATH="${PREFIX}/lib:${BUILD_PREFIX}/${HOST}/sysroot/usr/lib64:${LIBRARY_PATH}"
```

Might need something like this here to make sure the linker can find our libs when qmake runs. I've never used qmake, personally. I'm not sure if there is another qt-specific variable that could be set that would be the same thing. I think it's just calling gcc/ld eventually, though, so something like this _should_ work.

Reply:
Added it.